src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1357,11 +1361,14 @@ protected MembersAndInitializers GetMembersAndInitializers()
                 return membersAndInitializers;
             }
 
+            // Synthesized members need to be added in an independent builder to avoid modifying the one with declared members
+            var clone = getDeclaredMembersAndInitializers().Clone();
```

Comment:
>clone [](start = 16, length = 5)

I don't think that the fact that this was created by a clone operation is the most important thing about this local. Consider using more meaningful name. For example, "declaredAndSynthesizedMembersAndInitializers"